### ISSUE TITLE
ATL does not yet handle the case where Vorbis setup header spans across two OGG pages

### ISSUE THREAD

**Issue by JustArchi** [Zeugma440/atldotnet#35]
## The problem

I've tried to update `Comment` tag, part of `OpusTags` metadata in my sample `opus` audio file. The file is not updated with the new info, and after call to `Save()`, the content is replaced with previous data. See code to reproduce below for exact steps.

## Environment

Latest ATL version as of now (2.11.0), reproduced in `netcoreapp2.2` app, running in following environment:

```
 OS Name:     Windows
 OS Version:  10.0.17763
 OS Platform: Windows
```

## Details

Upon enabling logging:

```
ATL does not yet handle the case where Vorbis setup header spans across two OGG pages
```

Which is likely caused because of embedded cover art into the file. I'd appreciate if you could consider adding this functionality in the near future.

## Code To Reproduce Issue [ Good To Have ]

```c#
Track track = new Track("C:\\Path\\To\\sample.opus");
track.Comment = "Test";
track.Save();
```

I'm attaching file sample to let you reproduce the issue more easily.

[sample.zip](https://github.com/Zeugma440/atldotnet/files/3382903/sample.zip)

Thank you in advance!

**Comment by Zeugma440** [comment]
Heya @JustArchi and thanks for your interest in ATL :-)

I'll see what I can do. The sample is very valuable, as none of my test files had this very case.

Stay tuned !

PS : Kudos for ASF by the way; it's a must-have for any Steam user

**Comment by Zeugma440** [comment]
Looks like the issue isn't really what was displayed. I read the Opus spec a bit too fast and didn't realize Opus files do not completely rely on vorbis headers. 

The Opus comment header _does_ have the same format as the Vorbis comment header, but the big picture is a little different, as Opus fules actually have their own header structure.

**TL;DR** : I'm gonna write new code to update OPUS metadata properly

**Comment by Zeugma440** [comment]
Could you try with the latest commit of `master` branch and tell me if it works for you ?

If so, I'll release the fix on NuGet shortly

Thanks~

**Comment by JustArchi** [comment]
Hey.

Thanks a lot for fixing the issue! I can confirm that with master branch as of https://github.com/Zeugma440/atldotnet/commit/e83cf6eec1455d0a1f21f7e0b052f0a7b0513319 I'm no longer able to reproduce my initial issue, and at least the comments tag is now saved properly. I didn't find any sign of corruption or other problems with my `sample.opus` file above, therefore I think that this issue is entirely solved, at least for my own use case.

Thanks again! ❤️

**Comment by Zeugma440** [comment]
@JustArchi FYI, new NuGet package with the fix (v2.12) has just been published
